tests/test_base.py

```diff
@@ -168,3 +169,22 @@ def test_config_set(self):
                 DefFeature("Year", int, 1), DefFeature("Commits", int, 10)
             ),
         )
+
+@config
+class FakeTestingConfig2:
+    num: float
+    name: str = field("Name of FakeTesting2")
+    features: Features = Features(
+                DefFeature("default", int, 1), DefFeature("features", int, 10)
+            )
+    label: str = "unlabeled"
+
+@entrypoint("fake2")
+class FakeTesting2(BaseTesting):
+    
+    CONFIG = FakeTestingConfig2
+
+class TestCONFIG(unittest.TestCase):
+    def test_CONFIG(self):
+        with self.assertRaises(MissingConfig):
+            config = FakeTesting2.config({})
```

Comment:
We want to test regular instantiation, without the use of the `.config` method or `.withconfig`

```suggestion
            config = FakeTesting2()
```